client/plots/singleCellPlot.js

```diff
@@ -121,69 +121,63 @@ class singleCellPlot {
 							config: { settings: { singleCellPlot: { [`show${id}`]: e.target.checked } } }
 						})
 					})
-				headerDiv.append('label').text(plot.name).attr('for', `show${id}`)
+				topDiv.append('label').text(plot.name).attr('for', `show${id}`)
 			}
+		let selectCategory, violinBt, geneSearch, colorByGene, colorByPlot, searchboxDiv
 		if (q.singleCell?.geneExpression) {
-			searchGeneDiv.append('label').html('Gene expression:').style('padding', '10px')
-			const geneSearch = addGeneSearchbox({
+			headerDiv.append('label').text('Color by:').style('margin-right', '5px')
```

Comment:
please use `dom/radiobutton.js` to make radios which uses random id and is safe against conflict with external portals
on checking `Gene exp` radio button, apply focus on gene search box and no need for user having to click on it

Reply:
Used make_radios. Tried focus() on input but not working